mesa/agent.py

```diff
@@ -36,6 +37,81 @@ def step(self) -> None:
     def advance(self) -> None:
         pass
 
+    def forward_backward(self, amount, sign):
+        """Does the calculation to find  the agent's next move and is used within the forward and backward functions"""
+        new_x = float(self.pos[0]) + sign * math.cos(self.heading * math.pi / 180) * amount
+        new_y = float(self.pos[1]) + sign * math.sin(self.heading * math.pi / -180) * amount
+        next_move = (new_x, new_y)
```

Comment:
Should be `new_pos`. `next_move` implies the variable is an action, like "move up".